geniza/corpus/templates/corpus/snippets/document_transcription.html

```diff
@@ -115,14 +115,28 @@
                             <h2>{{ image_info.shelfmark }} {{ image_info.label }}</h2>
                             {% if not image_info.excluded %}
                                 {% if edit_mode %}
+                                    {# popout and close button #}
                                     <button class="primary popout-close-button" type="button" aria-label="Close"><i class="ph-x"></i></button>
                                     <button class="popout-button" type="button">
                                         <svg>
                                             <use xlink:href="{% static 'img/ui/desktop/all/pop-out.svg' %}#pop-out-icon" />
                                         </svg>
                                         <span class="sr-only">Pop out</span>
                                     </button>
+                                    <fieldset class="toolbar">
```

Comment:
Why are these added to the geniza template instead of to tahqiq?

I don't object necessarily, just want to understand

Reply:
I hadn't thought to do that! Was conceptualizing it as a Geniza feature so put it in Geniza, but I think it makes more sense to pass the toolbar element (in this case the fieldset) as a param to tahqiq, and instantiate the button elements there.